spec/classes/agent_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+Package = Puppet::Util::Package
```

Comment:
This line looks rather odd and I doubt that it's required

Reply:
That kind of hackish addition was just to keep the code under a reasonable number of characters per line but i guess after the changes to make the linter happy that ship has sailed anyway.